src/app/core/models/templates/tester-response-template.model.ts

```diff
@@ -8,28 +8,40 @@ export const TesterResponseHeaders = {
   teamResponse: new Header("Team's Response", 1),
   testerResponses: new Header('Items for the Tester to Verify', 1)
 };
+export const TeamAcceptedMessage = 'Your response not required for this bug as the team has accepted the bug as it is.';
+const TeamAcceptedRegex = new RegExp(TeamAcceptedMessage);
 
 export class TesterResponseTemplate extends Template {
   teamResponse: Section;
   testerResponse: TesterResponseSection;
   comment: IssueComment;
   teamChosenSeverity?: string;
   teamChosenType?: string;
+  teamAccepted?: boolean;
 
   constructor(githubIssueComments: GithubComment[]) {
     super(Object.values(TesterResponseHeaders));
 
     const templateConformingComment = githubIssueComments.find((comment) => this.test(comment.body));
-    if (templateConformingComment) {
-      this.comment = <IssueComment>{
-        ...templateConformingComment,
-        description: templateConformingComment.body
-      };
-      this.teamResponse = this.parseTeamResponse(this.comment.description);
-      this.testerResponse = this.parseTesterResponse(this.comment.description);
-      this.teamChosenSeverity = this.testerResponse.getTeamChosenSeverity();
-      this.teamChosenType = this.testerResponse.getTeamChosenType();
+    const teamAcceptedComment = githubIssueComments.find((comment) => this.testTeamAccepted(comment.body));
+
+    if (templateConformingComment === undefined && teamAcceptedComment === undefined) {
+      return;
+    }
+
+    if (teamAcceptedComment) {
+      this.teamAccepted = true;
+      return;
```

Comment:
It seems that we are breaking abstraction here by adding an additional `testTeamAccepted` method here when this template model just parses the team response template. 

Maybe we can consider abstracting out into the logic into another Child class which `extends Template`? Maybe something like a `TeamAcceptedTemplate`

Reply:
Ah ok makes sense